src/libhirte/common/cfg.c

```diff
@@ -63,24 +61,20 @@ static int parsing_handler(void *user, const char *section, const char *name, co
         }
         return 1;
 }
-#pragma GCC diagnostic pop
 
 int cfg_initialize(struct config **config) {
         struct config *new_cfg = malloc(sizeof(struct config));
         if (new_cfg == NULL) {
-                // error during configuration structure initialization
                 return -ENOMEM;
```

Comment:
If `malloc` fails, we are out of memory (`ENOMEM`) - I think the code is quite self-explanatory here. I'd reduce the use of inline comments as much as possible since it can cause quite a confusion when the code changes but the comments aren't updated (happens rather often).